index.js

```diff
@@ -34,7 +34,7 @@ var getFiles = function(options) {
 };
 
 var getOptions = function() {
-  var globalOptions = this.options.elm || {};
+  var globalOptions = this.options ? this.options.elm : {};
```

Comment:
since the `options` now became `query`, the code may look like:
```javascript
var globalOptions = this.options
    ? this.options.elm || {}
    : this.query.elm || {};
```